File: ElephantCarpaccio/Java/DemoTest.java
import org.junit.*;
import static org.junit.Assert.*;

public class DemoTest
{
    private boolean DEMO_TIME = true;

    @Test
    public void elephant_carpaccio_demo() 
    {
        if(!DEMO_TIME) return;

        ElephantCarpaccio.sliceIt();
    }
}

File: ElephantCarpaccio/C#/DemoTest.cs
using Moq;
using NUnit.Framework;

[TestFixture]
public class DemoTest
{
    private bool DEMO_TIME = true;

    [Test]
    public void elephant_carpaccio()
    {
        if(!DEMO_TIME) return;

        ElephantCarpaccio.SliceIt();
    }
}
